elasticsearch/example/src/main/java/example/springdata/elasticsearch/conference/EsConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.springdata.elasticsearch.conference;
+
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.elasticsearch.client.ClientConfiguration;
+import org.springframework.data.elasticsearch.client.RestClients;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+import org.springframework.data.elasticsearch.core.ElasticsearchRestTemplate;
+
+/**
+ * @author Prakhar Gupta
+ */
+@Configuration
+public class EsConfig {
```

Comment:
I wonder why this config is necessary as we rely on Spring Boot anyway that should provide a `ElasticsearchOperations`  bean.

Reply:
I thought it would be nice to add this config, where we would want to configure elastic search stuff on our own & not rely on Spring Boot auto-configuration. Since we already have example demonstrating auto-configuration in ```rest``` example.

Reply:
Removed this config